tests/api/rulesets.test.ts

```diff
@@ -14,15 +28,14 @@ describe('getRulesets()', () => {
       { user_id: 0, game_id: '', name: '', rules: {}, ruleset_id: 0 },
     ];
 
-    const mockDb = vi.fn().mockResolvedValueOnce(data);
-
-    // @ts-ignore
-    db.getRulesets = mockDb;
+    mocks.getRulesets.mockResolvedValueOnce(data)
 
     // 2. Execute code
     const result = await getRulesets(0, '');
 
     // 3. Assert results and expectations
     expect(result).toEqual(data);
+    expect(mocks.getRulesets).toHaveBeenCalledOnce()
+    expect(mocks.getRulesets).toHaveBeenLastCalledWith(0, '')
```

Comment:
Since we have a reference to the mocked function `getRulesets`, we can actually test if it was invoked at all, and we can see if it had the correct parameters as well.